shared-ts/disruptionTypes.zod.ts

```diff
@@ -785,6 +785,13 @@ export const consequenceSchema = z.discriminatedUnion("consequenceType", [
 
 export const MAX_CONSEQUENCES = 15;
 
+const historySchema = z.object({
```

Comment:
If this is identical to the historySchema that is in `disruption.schema.ts` then we should delete the schema in the `disruption.schema.ts` and update its usages to use the schema in the shared-ts folder to avoid duplicating code.

Reply:
done